<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCustomer</class>
 <widget class="QDialog" name="AddCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_companyName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_streetAddress">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_city">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_state">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_zip">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Company name *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Street Address *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_city">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>City *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_city_state">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>State *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_city_state_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ZIP *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key customer (1-0) * </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_key">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Interest">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
